sample_factory/algo/utils/action_distributions.py

```diff
@@ -206,7 +206,6 @@ def sample(self):
     def log_prob(self, actions):
         # split into batches of actions from individual distributions
         list_of_action_batches = torch.chunk(actions, len(self.distributions), dim=1)
-        list_of_action_batches = [a.squeeze(dim=1) for a in list_of_action_batches]
```

Comment:
We'll have to make sure that this does not break any of the environments.
VizDoom has multi-head action spaces in envs like Battle, this is where we can test it.